openwisp_monitoring/device/apps.py

```diff
@@ -11,3 +15,57 @@ class DeviceMonitoringConfig(AppConfig):
 
     def ready(self):
         manage_short_retention_policy()
+        from openwisp_controller.connection.models import DeviceConnection
```

Comment:
I'd move this code in its own method and make sure the feature can be disabled with a setting

Reply:
@nemesisdesign, I would have added an `if` condition prior to it, something like this
```
if getattr(settings, <setting-name>, True):
    # Rest of the code ... 
```
You have pointed it out correctly, this feature should have a setting attribute. Should I add it, something with name `OPENWISP_MONITORING_LISTEN_DEVICE_CONNECTION` ?